src/no/runsafe/clans/handlers/RankingHandler.java

```diff
@@ -1,16 +1,16 @@
 package no.runsafe.clans.handlers;
 
 import no.runsafe.clans.Clan;
-import no.runsafe.framework.api.IConfiguration;
-import no.runsafe.framework.api.event.plugin.IConfigurationChanged;
+import no.runsafe.clans.Config;
 
 import java.util.*;
 
-public class RankingHandler implements IConfigurationChanged
```

Comment:
The Config handles most config changes now, so RankingHandler doesn't need to implement IConfigurationChanged